src/components/app/BottomBox.css

```diff
@@ -102,7 +102,7 @@
   background-color: rgb(240 240 240 / 0.8);
   gap: 15px;
   inset: 0;
-  word-break: break-word;
+  overflow-wrap: break-word;
```

Comment:
This is a change from before. I should have used `overflow-wrap: anywhere` to keep the same behavior, but I think this value makes more sense.